src/domain/opportunity/opportunity.entity.ts

```diff
@@ -69,6 +70,17 @@ export class Opportunity extends BaseEntity
   )
   projects?: Project[];
 
+  @Field(() => [Aspect], {
+    nullable: true,
+    description: 'The set of solution aspects for this Opportunity',
+  })
+  @OneToMany(
+    () => Aspect,
+    aspect => aspect.opportunity,
+    { eager: true, cascade: true }
```

Comment:
Same comment as other PRs - do we want lazy loading?

Reply:
same comment as on actor groups / actor - lets first get it working and then can see re not using eager loading.